generated/nidaqmx/stream_readers.py

```diff
@@ -792,7 +792,7 @@ def read_many_sample(
         self._verify_array(voltage_data, number_of_samples_per_channel, False, True)
         self._verify_array(current_data, number_of_samples_per_channel, False, True)
 
-        _, samps_per_chan_read = self._interpreter.read_power_f64(
+        samps_per_chan_read = self._interpreter.read_power_f64(
```

Comment:
Sorry, my previous guidance was inconsistent. The hand-written read functions returned the read array, so let's continue to return the read array in the auto-generated read functions, as well as the few that you have hand-written.

It is important for the library interpreter and gRPC interpreter to agree about this.

https://github.com/ni/nidaqmx-python/pull/295#discussion_r1178529912

> [bkeryan](https://github.com/bkeryan) [2 days ago](https://github.com/ni/nidaqmx-python/pull/295#discussion_r1178529912)
> read_array is a mutable in/out parameter, so there is no need to return it.
>
>@[bkeryan](https://github.com/bkeryan) bkeryan [2 days ago](https://github.com/ni/nidaqmx->python/pull/295#discussion_r1178566623)
Nevermind, the hand-written functions did this too.

Reply:
I have updated these read functions to return `read_array`. Also, updated the callers too.